cmd/example-mysql/main.go

```diff
@@ -242,3 +245,22 @@ func parseTileIndexWidth(index string) (uint64, uint64, error) {
 
 	return i, w, nil
 }
+
+func initDatabaseSchema(ctx context.Context, db *sql.DB) {
+	if db != nil && *initSchemaPath != "" {
```

Comment:
I'd drop the nil db check as we know it isn't nil here, and if it was, it would be better to die quickly.

Reply:
A separate `*sql.DB` is initialized in the `initDatabaseSchema` method.